src/sbn_probability.cpp

```diff
@@ -358,14 +367,23 @@ double SBNProbability::ProbabilityOf(
   return exp(log_total_probability);
 }
 
-EigenVectorXd SBNProbability::ProbabilityOf(
+EigenVectorXd SBNProbability::ProbabilityOfCollection(
+    const EigenConstVectorXdRef sbn_parameters,
+    const std::vector<RootedIndexerRepresentation>& indexer_representations) {
+  // Lambdas aren't std:functions, so we make a std::function here.
```

Comment:
@EricJIsaac @ognian- 

Do you agree that a lambda can't be used here directly? I saw https://stackoverflow.com/questions/36030589/i-cannot-pass-lambda-as-stdfunction

The `EigenVectorXdOfStdVectorT` is defined in `eigen_sugar.hpp`.